### PR TITLE
Add usernames filter

### PR DESCRIPTION
**Pull request by markddavidoff**

### CODE REVIEW

File: slack_pull_reminder.py [hunk 1]
```diff
@@ -26,14 +33,19 @@
 
 
 def fetch_repository_pulls(repository):
-    return [pull for pull in repository.pull_requests()
-            if pull.state == 'open']
+    pulls = []
+    for pull in repository.pull_requests():
+        if pull.state == 'open':
+            if USERNAMES and pull.user.login.lower() in USERNAMES:
```
Comment (ekmartin) [inline slack_pull_reminder.py]: what happens if usernames is empty here? shouldn't it be:

```python
if pull.state == 'open' and (not USERNAMES or pull.user.login.lower() in USERNAMES):
  pulls.append(pull)
```

**Comment by ekmartin** [comment]
Out of curiosity, what would be a good use case for this?

File: slack_pull_reminder.py (hunk 1 above)
Comment (markddavidoff) [reply slack_pull_reminder.py]: good catch, my bad! fixed

**Comment by markddavidoff** [comment]
@ekmartin for our use case we have multiple teams using the same repos so we use this and the repos filter to only remind specific team members about their teams prs to specific repos

**Comment by ekmartin** [comment]
@markddavidoff: In that case, would it be more useful if you could provide _teams_ and have it filter by that? Team, as in https://github.com/orgs/ORGANIZATION/teams/

`TEAMS=team1, team2` -> retrieve team members for team1 and for team2 -> filter PRs by members in those teams

**Comment by markddavidoff** [comment]
Unfortunately we don't use GitHub teams, so for us that doesn't work. That would be neat to add but i like having usernames separate as well.

**Comment by markddavidoff** [comment]
@ekmartin Unfortunately we don't use GitHub teams, so for us that doesn't work. That would be neat to add but i like having usernames separate as well.

**Comment by markddavidoff** [comment]
Note:update doc

**Comment by markddavidoff** [comment]
@ekmartin added teams support in another PR but i think this is still useful such as for members not in a team that still might make prs that need to show up (architect, infosec, etc)

**Comment by markddavidoff** [comment]
@ekmartin any chance these features get in?

**Comment by ekmartin** [comment]
@markddavidoff: Thanks! Sorry about the late response.

**Pull request merged**